plugins/bitrise/README.md

```diff
@@ -0,0 +1,66 @@
+# Bitrise
+
+Welcome to the Bitrise plugin!
+
+- View recent Bitrise Builds for a Bitrise application
+- Download build artifacts
+
+## Installation
+
+```sh
+$ yarn add @backstage/plugin-bitrise
+```
+
+Then make sure to export the plugin in your app's [`plugins.ts`](https://github.com/backstage/backstage/blob/master/packages/app/src/plugins.ts) to enable the plugin:
+
+```js
+export { bitrisePlugin } from '@backstage/plugin-bitrise';
+```
+
+Modify your app routes in `App.tsx` to include the entry component exported from the plugin, for example:
+
+```tsx
+// At the top imports
+import { EntityBitrisePage } from '@backstage/plugin-bitrise';
+
+// Inside App component
+<Routes>
+  // ...
+  <Route path="/bitrise" element={<EntityBitrisePage />} />
+  // ...
+</Routes>;
+```
+
+Then configure the `plugin-bitrise` as described in the next section.
```

Comment:
...so I get here, now what?

If I'm not mistaken, the only way to use Bitrise is for me to hack the URL and type /bitrise, but I can't GET to it through the left nav sidebar or through any entity, is that right?

If that's the first run of it, totally fine. Might want to suggest that maybe in the doco, like this early release is exposed on that URL and you can decide what else to do with it, but many newer users might not know what else to do with it?

Reply:
@adamdmharvey I extended the docs now with more explicit instructions on how to add the plugin.

To answer your question: The plugin exposes a "Content" component that needs to be added to an Entity Page. It can only work in the context of an existing entity. It is not meant to be accessible through the nav sidebar.